pkg/storaged/luksmeta-monitor-hack.py

```diff
@@ -93,18 +93,15 @@ def info(dev):
         if in_luks2_token_section:
             match = re.match(b"  ([0-9]+): clevis$", line)
             if match:
-                try:
-                    token = subprocess.check_output(["cryptsetup", "token", "export", dev, "--token-id", match.group(1)],
-                                                    stderr=subprocess.PIPE)
-                    token_object = json.loads(token.decode("utf-8"))
-                    if token_object.get("type") == "clevis":
-                        config = json.dumps(get_clevis_config_from_protected_header(token_object["jwe"]["protected"]))
-                        for slot_str in token_object.get("keyslots", []):
-                            slot = int(slot_str)
-                            slots[slot] = {"Index": {"v": slot},
-                                           "ClevisConfig": {"v": config}}
-                except subprocess.CalledProcessError:
-                    pass
+                result = subprocess.run(["cryptsetup", "token", "export", dev, "--token-id", match.group(1)],
+                                        check=False, capture_output=True, universal_newlines=True)
+                token_object = json.loads(result.stdout)
```

Comment:
This looks broken, or at least dubious. With `check=False`, we need to validate whether calling the command was successful before processing its output. This may or may not be the cause of [this regression](https://logs.cockpit-project.org/logs/pull-16944-20220420-122609-856b7ebc-rhel-8-6-distropkg/log.html#184-2).

However, I kinda disagree that this was out of style -- catching exceptions and assuming silent success is Pythonic.

Reply:
See below, backed out.